libs/components/indicators/src/lib/modules/wait/wait.component.ts

```diff
@@ -104,13 +114,30 @@ export class SkyWaitComponent implements OnInit, OnDestroy {
     return this.#_isNonBlocking;
   }
 
+  /**
+   * Screen reader text [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility) when the wait toggles off.
+   *  The default varies based on whether the wait is for an element or a page.
+   * For example, the default for a page-level wait is "Page loading complete."
+   * For element-level waits, we recommend that consumers overwrite the default to describe the specific element.
+   * For more information, see the Design tab and the [WCAG documentation on status messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html).
+   */
+  @Input()
+  public set screenReaderCompletedText(value: string | undefined) {
+    this.#customScreenReaderCompletedText = value;
+    this.#publishScreenReaderCompletedText();
+  }
+
   public ariaLabelStream = new BehaviorSubject<string>('');
+  public ariaLiveText = '';
+  public screenReaderCompletedTextStream = new BehaviorSubject<string>('');
 
   #customAriaLabel: string | undefined;
+  #customScreenReaderCompletedText: string | undefined;
```

Comment:
I only see this private field referenced in the `screenReaderCompletedText` and the `#publishScreenReaderCompletedText()` function. You can get rid of this private field and just pass the value directly into the `#publishScreenReaderCompletedText()` function which will make the code flow easier to follow.

Reply:
I don't think we can do that - because `publishScreenReaderCompletedText` could be called later after the time that the `screenReaderCompletedText` setter is called. For example - someone could set up the completed text and then could switch a wait to be `isFullPage` and so the `publishScreenReaderCompletedText` would be called again and would need the value that is saved in this field.

Reply:
Done